.github/workflows/_25_package.yml

```diff
@@ -35,109 +35,6 @@ jobs:
-        shell: bash
-        run: |
-          git config --global --add safe.directory '*'
-          TARGET_BRANCH=${{ env.TARGET_BRANCH }}
-          echo "The target branch is $TARGET_BRANCH"
-          VERSION=$(echo "$TARGET_BRANCH" | grep -oE '[0-9]+\.[0-9]+')
-          MAJOR=$(echo $VERSION | cut -d. -f1)
-          MINOR=$(echo $VERSION | cut -d. -f2)
-
-          # Compute the previous version
-          if [ $MINOR -eq 0 ]; then
-            PREV_MAJOR=$((MAJOR - 1))
-            if [ $PREV_MAJOR -ge 0 ]; then
-              PREV_VERSION="${PREV_MAJOR}"
-            else
-              echo "No previous version available"
-              exit 1
-            fi
-          else
-            PREV_MINOR=$((MINOR - 1))
-            PREV_VERSION="${MAJOR}.${PREV_MINOR}"
-          fi
-
-          echo "Previous version: $PREV_VERSION"
-
-          if [[ "$PREV_VERSION" == *.* ]]; then
-            # Find the highest tag matching the previous minor version prefix
-            HIGHEST_TAG=$(git tag -l "${PREV_VERSION}.*" | sort -V | tail -n 1)
-          else
-            # Find the highest tag for the previous major version
-            HIGHEST_TAG=$(git tag -l "${PREV_VERSION}.*.*" | sort -V | tail -n 1)
-          fi
-
-          if [ -z "$HIGHEST_TAG" ]; then
-            echo "No tags found for previous version $PREV_VERSION"
-            exit 1
-          fi
-
-          echo "HIGHEST_TAG=$HIGHEST_TAG"
-          PREVIOUS_RELEASE_COMMIT=$(git rev-list -n 1 $HIGHEST_TAG)
-          echo "PREVIOUS_RELEASE_COMMIT=$PREVIOUS_RELEASE_COMMIT"
-          echo "PREVIOUS_RELEASE_COMMIT=$PREVIOUS_RELEASE_COMMIT" >> $GITHUB_ENV
-
-      # If we're on a release branch we need to download the previous release binaries
-      - name: Download latest release binaries for release branch.
-        if: env.IS_RELEASE_BRANCH == 'true'
-        uses: dawidd6/action-download-artifact@e7466d1a7587ed14867642c2ca74b5bcc1e19a2d
-        with:
-          workflow: release-berghain.yml
-          name: chainflip-backend-bin
-          commit: ${{ env.PREVIOUS_RELEASE_COMMIT }}
-          github_token: ${{ secrets.CF_BACKEND_GITHUB_TOKEN }}
-          path: latest-release
-
-      # If we're on main then the latest run of the release-sisyphos.yml workflow will have the latest release binaries
-      # Therefore we don't need to provide a commit.
-      - name: Download latest release binaries for main/PR branches.
-        if: env.IS_RELEASE_BRANCH == 'false'
-        uses: dawidd6/action-download-artifact@e7466d1a7587ed14867642c2ca74b5bcc1e19a2d
-        with:
-          workflow: release-sisyphos.yml
-          name: chainflip-backend-bin
-          github_token: ${{ secrets.CF_BACKEND_GITHUB_TOKEN }}
-          path: latest-release
-
-      # See: engine-runner-bin/Cargo.toml for more information
-      - name: Move latest release dylib into the target/release ready to be packaged by cargo-deb.
-        run: |
-          mv latest-release/*.so target/release/
```

Comment:
We can delete this and all of the above because the `engine-runner` build script now downloads it the old dylib.